system/jhtree/keydiff.cpp

```diff
@@ -710,13 +710,13 @@ class KeyDiffStats
     void addDiffSize(size32_t sz) { diffSize += sz; }
     void log() const
     {
-        LOG(MCstats, "Matching rows: %u", stats[CKeyDiff::CMD_MATCH-1]);
-        LOG(MCstats, "Rows close to previous old row: %u", stats[CKeyDiff::CMD_DIFF_OLD_PREV-1]);
-        LOG(MCstats, "Rows close to current old row: %u", stats[CKeyDiff::CMD_DIFF_OLD_CURR-1]);
-        LOG(MCstats, "Rows close to previous new row: %u", stats[CKeyDiff::CMD_DIFF_NEW_PREV-1]);
+        LOG(MCoperatorWarning, "Matching rows: %u", stats[CKeyDiff::CMD_MATCH-1]);
+        LOG(MCoperatorWarning, "Rows close to previous old row: %u", stats[CKeyDiff::CMD_DIFF_OLD_PREV-1]);
+        LOG(MCoperatorWarning, "Rows close to current old row: %u", stats[CKeyDiff::CMD_DIFF_OLD_CURR-1]);
+        LOG(MCoperatorProgress, "Rows close to previous new row: %u", stats[CKeyDiff::CMD_DIFF_NEW_PREV-1]);
```

Comment:
MCStats was replaced by MCoperatorProgress, assuming that's what you meant (not MCoperatorInfo). Let me know, if you wanted to change it to Info.

Reply:
It is fine to leave it, but I in this case MCoperatorInfo would be the correct category.  An example where using an alias for a category confuses rather than helps.